client/src/components/GeoIdSection.vue

```diff
@@ -0,0 +1,57 @@
+<template>
+  <div class='selected' v-if='selected'>
```

Comment:
You can [bind class to a variable](https://vuejs.org/guide/essentials/class-and-style.html#binding-html-classes) instead of having separate divs for this.

Reply:
thanks!